Makefile

```diff
@@ -308,7 +308,7 @@ system-test-deps:
 	$(MAKE) system-smoke-test-deps
 	docker pull "aeternity/aeternity:v2.1.0"
 	docker pull "aeternity/aeternity:v2.3.0"
-	docker pull "aeternity/aeternity:v3.1.0"
```

Comment:
This version is updated because the new major release is not a hard fork and therefore no extra image is needed?

Reply:
This is not about hard fork (re consensus protocol) but about channels APIs i.e. the node-to-client-WebSocket and the node-to-node-Noise.

Before this commit, this pull of v3.1.0 was unused as the corresponding tests were commented out.

The tests meant to use this line test that the version of the node built from the codebase passes a smoke test re compatibility of channel APIs (WebSocket and Noise) with the oldest compatible release. That is 4.0.0 hence the change.

Reply:
Got it. Thanks.